<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_dev</class>
 <widget class="QWidget" name="change_dev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>28</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="electr">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>77</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="gas">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>122</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="water">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>170</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>56</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>102</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>148</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>198</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="enter">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>279</y>
     <width>173</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>279</y>
     <width>173</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>234</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>345</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pich.qrc">:/picha/Pichures/change_dev.png</pixmap>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>name</zorder>
  <zorder>electr</zorder>
  <zorder>gas</zorder>
  <zorder>water</zorder>
  <zorder>warning</zorder>
  <zorder>warning_2</zorder>
  <zorder>warning_3</zorder>
  <zorder>warning_4</zorder>
  <zorder>enter</zorder>
  <zorder>back</zorder>
  <zorder>warning_5</zorder>
 </widget>
 <resources>
  <include location="pich.qrc"/>
 </resources>
 <connections/>
</ui>
